instrumentation/net/http/go.mod

```diff
@@ -7,7 +7,7 @@ replace go.opentelemetry.io/contrib => ../../..
 require (
 	github.com/felixge/httpsnoop v1.0.1
 	github.com/stretchr/testify v1.6.1
-	go.opentelemetry.io/contrib v0.10.0
+	go.opentelemetry.io/contrib v0.10.1
```

Comment:
Can you remove the `replace` directive in this file?  See #208.

Reply:
Should this be done for all the `go.mod` files? It seems like they all have `replace` directives for the contrib repository.

IIRC this was something we did to bootstrap, but I also wondered if we should keep them if we want to change local dependencies in the repo and refer to them. I guess we could add them back if that is the case, right?

Also, will it not break things if I remove this `replace` before I actually do the tag and release? I.e. there won't actually be a `v0.10.1` release yet.

Reply:
Maybe this is a lapse in understanding how `go get` works, but wont the `replace` directive break things if a user calls `go get` on the particular integration?

But I agree that making the change before will temporarily break things

Reply:
All good points.  Perhaps this is something we can discuss at the SIG meeting today.  We should probably  be safe and not change anything until after a release.